bridges/HardwareInfoBridge.php

```diff
@@ -0,0 +1,66 @@
+<?php
+class HardwareInfoBridge extends FeedExpander
+{
+	const NAME = 'Hardware Info Bridge';
+	const URI = 'https://HardwareInfo.net/';
+	const DESCRIPTION = 'Tech news from hardware.info (Dutch)';
```

Comment:
hardware.info or hardware.net? First does not resolve for real, second has problems with certificate.

Reply:
https://nl.hardware.info/

Reply:
https://hardware.info/ , which redirects to https://nl.hardware.info/

Reply:
fixed